src/networking.c

```diff
@@ -399,6 +399,25 @@ void addReplyErrorLength(client *c, const char *s, size_t len) {
 
 /* Do some actions after an error reply was sent (Log if needed, updates stats, etc.) */
 void afterErrorReply(client *c, const char *s, size_t len) {
+    struct redisError *err = NULL;
+    /* Errors that occurred during command processing */
+    if (c->cmd){
+        c->cmd->failed_calls++; /* Update per command error stats */
+        err = lookupError(shared.commanderror);
+        if (err) err->count++;
+    }
+    /* Authorization error counter (commands blocked by ACLs) */
+    if (!strncmp(s, "-NOPERM", 7)){
+        err = lookupError(shared.authorizationerror);
+        if (err) err->count++;
+    /* Authentication errors */
+    } else if (!strncmp(s, "AUTH", 4) ||
+               !strncmp(s, "-NOAUTH", 7) ||
+               !strncmp(s, "-WRONGPASS", 10)){
+        err = lookupError(shared.authenticationerror);
```

Comment:
Can this function just increment the error? Not sure why we need to fetch it then increment it.

Reply:
agree @madolson . changed in the latest commit